src/cudadecoder/Makefile

```diff
@@ -13,8 +13,12 @@ endif
 
 TESTFILES =
 
-OBJFILES = batched-threaded-nnet3-cuda-pipeline.o decodable-cumatrix.o \
-           cuda-decoder.o cuda-decoder-kernels.o cuda-fst.o
+OBJFILES = cuda-decoder.o cuda-decoder-kernels.o cuda-fst.o \
+	   batched-threaded-nnet3-cuda-online-pipeline.o \
+	   batched-threaded-nnet3-cuda-pipeline.o \
+	   batched-threaded-nnet3-cuda-pipeline2.o \
+	   batched-static-nnet3.o batched-static-nnet3-kernels.o \
+	   deprecated/decodable-cumatrix.o
```

Comment:
Hm, I'm concerned that by messing with the standard directory structure, we may make maintaining the CMake and Windows builds harder.

Reply:
Ok, I'll switch it back to the cudadecoder/ directory with some comments. We'll remove those files in a few months